cloud/blockstore/tools/csi_driver/client/main.go

```diff
@@ -189,6 +189,70 @@ func newDeleteVolumeCommand(endpoint *string) *cobra.Command {
 	return &cmd
 }
 
+func newNodeStageVolumeCommand(endpoint *string) *cobra.Command {
+	var volumeId, stagingTargetPath string
+	cmd := cobra.Command{
+		Use:   "stagevolume",
+		Short: "Send stage volume request to the CSI node",
+		Run: func(cmd *cobra.Command, args []string) {
+			ctx, cancelFunc := context.WithTimeout(
+				context.Background(),
+				120*time.Second,
+			)
+			defer cancelFunc()
+			client, err := newNodeClient(ctx, *endpoint)
+
```

Comment:
Extra line break, err  != nil is usually padded towards the call which created the error.

Reply:
Fixed here and in other places